docs/detections/machine-learning/machine-learning.asciidoc

```diff
@@ -76,6 +80,19 @@ prior to the time they are enabled. After jobs are enabled, they continuously
 analyze incoming data. When jobs are stopped and restarted within the two-week
 time frame, previously analyzed data is not processed again.
 
+[float]
+[[ml-integrations]]
+=== Jobs in advanced analytics (UEBA) Elastic integrations
```

Comment:
Are we capitalizing `Advanced Analytics`? 
```suggestion
=== Jobs in advanced analytics (UEBA) Elastic integrations
```

Reply:
Done, thank you!